MonoGame.Framework/Media/MediaPlayer.cs

```diff
@@ -69,17 +109,40 @@ private set
                     EventHelpers.Raise(null, MediaStateChanged, EventArgs.Empty);
                 }
             }
-        }
-
+        }
+
+        /// <summary>
+        /// Determines whether the game has control of the background music.
+        /// </summary>
+        /// <remarks>
+        /// A gamer can play their own music as the background to your game by using the Xbox 360 dashboard.
```

Comment:
We will probably want to keep an eye out for references to things like Xbox 360/Zune or Windows Phone. Most of those specific comments will no longer be needed. We've had to do this with other documentation we are importing for the tutorials side of things.